download/download.adoc

```diff
@@ -115,6 +111,6 @@ For more info about the Drools Docker images see this http://blog.athico.com/201
 
 === Eclipse update site
 
-* https://download.jboss.org/drools/release/#{site.pom.latestFinal.version}/org.drools.updatesite/[Click here] to go to the Drools and jBPM update site #{site.pom.latestFinal.version}.
+* https://download.jboss.org/drools/release/7.48.0.Final/org.drools.updatesite/[Click here] to go to the Drools and jBPM update site (latest released version *7.48.0.Final*).
```

Comment:
we should add here a note that we stopped synchronously release the Eclipse tools with the Drools releases

Reply:
![image](https://user-images.githubusercontent.com/3669059/106887699-d3aff480-66e5-11eb-9ea2-2b0a16f02764.png)